visualization/app/codeCharta/util/fileValidator.ts

```diff
@@ -35,41 +35,57 @@ export const ERROR_MESSAGES = {
 export function validate(nameDataPair: NameDataPair) {
```

Comment:
I do not think `validate()` needs to be exported.

Reply:
Implemented in the next commit.